src/lib/util.ts

```diff
@@ -2,11 +2,14 @@ import chalk from 'chalk';
 import commandLineUsage, { Section } from 'command-line-usage';
 import figlet from 'figlet';
 import path from 'path';
+import npm from 'npm-programmatic';
+// import { exec } from 'child_process';
```

Comment:
Do we need this commented out line? @ttaylor92

Reply:
corrected